exercises/concept/calculator-conundrum/.docs/hints.md

```diff
@@ -0,0 +1,15 @@
+# Hints
+
+### 1. Implementing operations
+
+* Use a `switch-case` block to match different operations and implement them using `SimpleOperation` class.
+
+### 2. Handling invalid operation
+
+* Check for `null` operation beforehand and the rest along-with the `switch-case` block.
```

Comment:
I think this first bullet point should probably be split:

* Check for `null` operations before the switch.
* Add a case to the switch for the empty String.
* Use a default for anything else at the end of the switch.